api/libproject/generate_appengine_web_xml.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+set -euo pipefail
+IFS=$'\n\t'
+
+CIRCLECI="${CIRCLECI:-false}"
+
+if [[ ! "$CIRCLECI" == "true" ]]; then
+  cd db
+  source vars.env
+  cd ..
+else
+  WORKBENCH_DB_PASSWORD="$WORKBENCH_PASSWORD"
```

Comment:
Definitely. I didn't change it because I don't think I have access to that value, so I can't update the environment variable in Circle. Do you have it?